src/resource/tiles/tiles.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { Controller, Get, Post, Body, Param, Patch, Delete } from "@nestjs/common";
+import { TilesService } from "./tiles.service";
+import { Tile } from "./schemas/tile.schema";
+import { CreateTileDto } from "./dto/create-tile.dto";
+import { UpdateTileDto } from "./dto/update-tile.dto";
+
+@Controller("tiles")
+export class TilesController {
+  constructor(private readonly tilesService: TilesService) {}
+  @Get()
+  async findAll(): Promise<Tile[]> {
+    return await this.tilesService.findAll();
+  }
+
+  @Get(":id")
+  async findOne(@Param("id") id: string): Promise<Tile> {
+    return await this.tilesService.findOne(id);
+  }
+
+  @Post()
+  async create(@Body() createTileDto: CreateTileDto): Promise<Tile> {
+    return await this.tilesService.create(createTileDto);
+  }
+
+  @Patch(":id")
```

Comment:
use `Put` instead

Reply:
Did you actually fix this issue before you click resolve conversation?